deployment/samples/docker-compose/docker-compose.yml

```diff
@@ -7,8 +7,8 @@ volumes:
 services:
   pyaleph:
     restart: always
-    image: alephim/pyaleph-node:v0.2.1
-    command: pyaleph --config /opt/pyaleph/config.yml --key-dir /opt/pyaleph/keys -v
+    image: alephim/pyaleph-node:latest
```

Comment:
The doc points to a fixed version since release 0.2.0. We usually bump the version number on each release. The point of changing here is that anyway, this file will not work anymore with v0.2.1 because of the entrypoint change.

Reply:
I understand. Annoying but makes sense.